examples/layers/services/media_service.dart

```diff
@@ -42,7 +42,7 @@ class PianoKey {
   Future<Null> load(mojom.MediaServiceProxy mediaService) async {
     try {
       mediaService.createPlayer(player);
-      await player.prepare(await http.readDataPipe(soundUrl));
+      player.prepare(await http.readDataPipe(soundUrl), (bool ignored) { });
```

Comment:
you can't pass null?

we should send feedback upstream saying that null should be supported as a no-op callback